spoke-contract/README.md

```
@@ -0,0 +1,7 @@
+Spoke-owned PR docs check (not loaded from the Hub).
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)